packages/js/product-editor/src/components/add-new-shipping-class-modal/add-new-shipping-class-modal.tsx

```diff
@@ -40,15 +38,12 @@ function ShippingClassForm( { onAdd, onCancel }: ShippingClassFormProps ) {
 			<TextControl
 				{ ...getInputProps( 'name' ) }
 				placeholder={ __( 'e.g. Fragile products', 'woocommerce' ) }
-				label={ interpolateComponents( {
-					mixedString: __( 'Name {{required/}}', 'woocommerce' ),
-					components: {
-						required: (
-							<span className="woocommerce-add-new-shipping-class-modal__optional-input">
-								{ __( '(required)', 'woocommerce' ) }
-							</span>
-						),
-					},
+				label={ createInterpolateElement( 'Name <required />', {
```

Comment:
I think we'll need to make this string translatable:

```suggestion
				label={ createInterpolateElement( __( 'Name <required />', 'woocommerce' ), {
```

Reply:
Nice catch! Done in 442b9746cfa0d1623759860d9131326e2dec810a.